sdk/appcenter/src/test/java/com/microsoft/appcenter/AbstractAppCenterServiceTest.java

```diff
@@ -92,6 +92,14 @@ public Object answer(InvocationOnMock invocation) {
         SharedPreferencesManager.putBoolean(eq(SERVICE_ENABLED_KEY), anyBoolean());
     }
 
+    @Test
+    public void onApplicationStarted() {
+        mService.onApplicationStarted();
+    }
```

Comment:
What does it test?

Reply:
Coverage fix

Reply:
Did you verify subscription on these events? Please cover this part properly